src/sass/components/_about.scss

```
@@ -0,0 +1,22 @@
+.about {
+  max-width: $mobile-width;
+  margin: 0 auto;
+  padding: 52px 0 52px 0;
```

FYI you can use shorthand notation in this case
```suggestion
  padding: 52px 0;
```